modules/sdk-coin-avaxp/src/lib/transaction.ts

```diff
@@ -222,10 +222,15 @@ export class Transaction extends BaseTransaction {
       case TransactionType.Import:
         return (this.avaxPTransaction as ImportTx | EVMImportTx).getOuts().map(utils.mapOutputToEntry(this._network));
       case TransactionType.Export:
-        return (this.avaxPTransaction as ExportTx).getExportOutputs().map(utils.mapOutputToEntry(this._network));
+        if (utils.isTransactionOf(this._avaxpTransaction, this._network.cChainBlockchainID)) {
+          return (this.avaxPTransaction as EVMExportTx).getExportedOutputs().map(utils.mapOutputToEntry(this._network));
+        } else {
+          return (this.avaxPTransaction as ExportTx).getExportOutputs().map(utils.mapOutputToEntry(this._network));
+        }
+      case TransactionType.AddDelegator:
```

Comment:
@lcovar this case needed to make an AddDelegator. But you need to add a unit test to ensure that it works with AddDelegatorTx.